README.md

```diff
@@ -47,6 +63,11 @@ If you prefer to [compile Webots from source](https://github.com/cyberbotics/web
 
 ### About us
 
-Webots was originally designed at [EPFL](https://epfl.ch) in 1996 and then further developed and commercialized by [Cyberbotics](https://cyberbotics.com) since 1998. In December 2018, Webots was open sourced. Since then, [Cyberbotics](https://cyberbotics.com) continues to develop Webots thanks to paid customer support, training, consulting for industry and academic research projects.
+Webots was originally designed at [EPFL](https://epfl.ch) in 1996 and then further developed and commercialized by [Cyberbotics](https://cyberbotics.com) since 1998. Originally it was meant to be used as a research tool to investigate various control algorithms in mobile robotics. In December 2018, Webots was open sourced. Since then, [Cyberbotics](https://cyberbotics.com) continues to develop Webots thanks to paid customer support, training, consulting for industry and academic research projects. We specialize in the development, rendering, and representation of robotic, mechanical, and vehicle systems through our application. Our many different uses include sensor-equiped autonomous vehicles, real-time image segmentation, fluid simulations for marine research, and much much more! 
```

Comment:
This is a bit redundant. Here is my suggestion:
```suggestion
Webots was originally designed at [EPFL](https://epfl.ch) in 1996 and then further developed and commercialized by [Cyberbotics](https://cyberbotics.com) since 1998. Originally, it was meant to be used as a research tool for mobile robotics. In December 2018, Webots was open sourced. Since then, [Cyberbotics](https://cyberbotics.com) continues to develop Webots thanks to paid customer support, training, consulting for industry and academic research projects.
```